addons/hr_timesheet_sheet/migrations/9.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,22 @@
+---Fields in module 'hr_timesheet_sheet'---
+
+hr_timesheet_sheet / hr_timesheet_sheet.sheet / message_follower_ids (many2many): not a function anymore
+hr_timesheet_sheet / hr_timesheet_sheet.sheet / message_follower_ids (many2many): relation is now 'mail.followers' ('res.partner')
+hr_timesheet_sheet / hr_timesheet_sheet.sheet / message_follower_ids (many2many): type is now 'one2many' ('many2many')
+# TODO : ???  # Concerns module Mail (has to be done in another migration script specifically for that module) ---> Nothing to do
+
+hr_timesheet_sheet / hr_timesheet_sheet.sheet / timesheet_ids (one2many)      : relation is now 'account.analytic.line' ('hr.analytic.timesheet')
+#Nothing To Do : the records will be retrieved automatically as hr.analytic.timesheet inherits 'account.analytic.line'
```

Comment:
Well, you will still need to move the field from hr_analytic_timesheet to account_analytic_line (the analysis does not mention it here, but it is implied by the fact that the model hr.analytic.timesheet is removed, as mentioned in https://github.com/OCA/OpenUpgrade/blob/9.0/openerp/addons/base/migrations/9.0.1.3/openupgrade_general_log.txt#L128).